packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/share/requests/ShareLinkCreateRequest.java

```diff
@@ -2,31 +2,46 @@
 
 import cloud.pangeacyber.pangea.BaseRequest;
 import cloud.pangeacyber.pangea.share.models.ShareLinkCreateItem;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.List;
 
 public class ShareLinkCreateRequest extends BaseRequest {
 
+	/** The bucket to use, if not the default. */
+	@JsonInclude(Include.NON_NULL)
+	@JsonProperty("bucket_id")
+	private String bucketId;
+
 	@JsonProperty("links")
 	private List<ShareLinkCreateItem> links;
 
 	private ShareLinkCreateRequest(Builder builder) {
 		this.links = builder.links;
+		this.bucketId = builder.bucketId;
 	}
 
 	public List<ShareLinkCreateItem> getLinks() {
 		return links;
 	}
 
 	public static class Builder {
-
```

Comment:
nitpick, but I think prettier-java wants a blank newline after the start of the class? As long as you got this from`mvn prettier:write` then it's all good.